cymetric/convenient_interface.py

```diff
@@ -0,0 +1,650 @@
+#!/usr/bin/env python
+
+import warnings
+
+import pandas as pd
+import numpy as np
+
+import cymetric as cym
+from cymetric import tools
+
+try:
+    from pyne import data
+    import pyne.enrichment as enr
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
+
+try:
+    from graphviz import Digraph
+    HAVE_GRAPHVIZ = True
+except ImportError:
+    HAVE_GRAPHVIZ = False
+
+
+def format_nuclist(nuc_list):
+    """
+    format the nuclide list provided by the users into a standart format:
+    ZZAAIIIII.
+    Parameters
+    ----------
+    nuc_list:  list of nuclides
+    """
+    tools.raise_no_pyne('Unable to format nuclide list!', HAVE_PYNE)
+
+    for i in range(len(nuc_list)):
+        nuc_list[i] = nucname.id(nuc_list[i])
+
+    return nuc_list
```

Comment:
In Python, you can iterate through a list without indexing. I would change the above to a single list comprehension:

```python
return [nucname.id(nuc) for nuc in nuc_list]
```